src/lib/components/events/event-carousel.svelte

```diff
@@ -15,6 +16,7 @@
 			left: -movementScalar,
 			behavior: isSmooth ? 'smooth' : 'auto',
 		});
+	var hasHorizontalScrollbar = false;
```

Comment:
This variable is best declared with `let` instead of `var`.

Reply:
Done.